core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -398,6 +398,12 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
     @Setter
     private Entity mouseoverEntity;
 
+    /**
+     * Stores all Java recipes by identifier, and matches them to all possible Bedrock recipe identifiers.
+     */
+    @Setter @Getter
+    private Map<String, List<String>> identifierToBedrockRecipes;
```

Comment:
The getter on this field is superfluous because the getter annotation on the class level applies it to every single field

Reply:
Addressed, ty